README.md

```diff
@@ -38,3 +41,28 @@ Please note that on Mac, once a private key is used with a certificate, that cer
 ```
 static: certificate has an existing certificate-key pair that was previously imported into the Keychain. Using key from Keychain instead of the one provided.
 ```
+
+### OpenSSL and LibCrypto (Unix only)
```

Comment:
Move this up above the `Mac-Only TLS Behavior` section

This openssl stuff is relevent to building/installation, it should be close to the existing build/install steps